tests/src/utils/gameStateUtil.test.ts

```diff
@@ -100,7 +100,7 @@ testSingletonModule('utils/gameStateUtil', ({ testUnit }) => {
 
     testScenario('when given an invalid game state')
       .arrange(() => {
-        sinon.stub(databaseUtil, 'save');
+        sinon.stub(databaseUtil, 'save').returns(true);
```

Comment:
Is this stub needed? Shouldn't it `throw` before calling `databaseUtil.save()`

Reply:
The test doesn't know that and it needs to be stubbed for the assertion

Reply:
Hmmm... I guess if the code got to `databaseUtil.save()` for some reason, the test's `assert` would still fail.